### ISSUE TITLE
Define form factor

### ISSUE THREAD

**Issue by SeanAlling** [EE-Emerge/EE-Emerge2021_PCB#2]
What shape shall the PCB take? What are the design constraints?

**Comment by ScottyMcCotty** [comment]
I don't know (and haven't gone to check) which fan we chose, but one design constraint is the size of the fan. Ideally the PCB would have the same dimensions as the fan so that they can stack together without having to make weird PCB protrusions out of our box. There's no point in designing the PCB to be less wide or less long than the fan, since the enclosure will be at least large enough for the fan.

**Comment by SeanAlling** [comment]
Where will the fan e in relation to the PCB? The PCB can't block the fan otherwise that can cause air flow issues. 

I think the fan and PCB can be 90 degrees to one another. Then the PCB can really be any size but like you mention the width is still limited by the width of the fan. 

Depending on how the enclosure is selected (3D printed or off the shelf), the form factor and shape of the PCB will depend on the size of the fan and the shape of the enclosure. Obviously 3D printing the enclosure means we can adjust the enclosure to fit the PCB instead of the other way around.

**Comment by ScottyMcCotty** [comment]
You're right, having the PCB plug the fan would be bad. Having them form an L shape with the fan attached to (can it be screwed?) the PCB would define all 3 dimensions of the box.

I guess it would make sense to find the dimensions of the fan, estimate the smallest size we could make the PCB, then look for off-the-shelf enclosures which are bigger than the smallest size. Then the PCB can be re-sized to fit the enclosure

**Comment by ryankim2731** [comment]
Not sure how much of a factor this could play, but would the vibrations caused by the fan possibly damage the circuit over time? Another possible restraint to the PCB is aiming for the lightest weight, so having a 2 layer PCB would probably be the best in terms of weight.
